README.md

```diff
@@ -1,69 +1,115 @@
+
+### Using pip
 
 ```sh
 pip3 install minio
 ```
 
-## Download source
+### From GitHub
 
 ```sh
 git clone https://github.com/minio/minio-py
 cd minio-py
 python setup.py install
 ```
 
-## Quick Start Example - File Uploader
-This example program connects to an S3-compatible object storage server, make a bucket on that server, and upload a file to the bucket.
-
-You need the following items to connect to an S3-compatible object storage server:
+## Initialize a MinIO Client Object
 
-| Parameters | Description                                                |
-|------------|------------------------------------------------------------|
-| Endpoint   | URL to S3 service.                                         |
-| Access Key | Access key (aka user ID) of an account in the S3 service.  |
-| Secret Key | Secret key (aka password) of an account in the S3 service. |
+The MinIO client requires the following parameters to connect to an Amazon S3 compatible object storage:
 
-This example uses MinIO server playground [https://play.min.io](https://play.min.io). Feel free to use this service for test and development.
+| Parameter  | Description                                            |
+|------------|--------------------------------------------------------|
+| Endpoint   | URL to S3 service.                                     |
+| Access Key | Access key (user ID) of an account in the S3 service.  |
+| Secret Key | Secret key (password) of an account in the S3 service. |
 
-### file_uploader.py
 ```py
 from minio import Minio
 from minio.error import S3Error
 
 
+def main():
+    # Create a client with the MinIO server playground, its access key          
+    # and secret key.                                                           
+    client = Minio(
+        endpoint="play.min.io",
+        access_key="Q3AM3UQ867SPQQA43P2F",
+        secret_key="zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG",
+    )
+
+    print("MinIO Python SDK client initialized", client)
+
+if __name__ == "__main__":
+    try:
+        main()
+    except S3Error as exc:
+        print("error occurred.", exc)
+```
+
+## Example - File Uploader
+
+This sample code connects to an object storage server, creates a bucket, and uploads a file to the bucket.
+It uses the MinIO `play` server, a public MinIO cluster located at [https://play.min.io](https://play.min.io).
+
+The `play` server runs the latest stable version of MinIO and may be used for testing and development.
+The access credentials shown in this example are open to the public and all data uploaded to `play` should be considered public and non-protected.
+
+### `file_uploader.py`
+
+This example does the following:
+
+- Connects to the MinIO `play` server using the provided credentials.
+- Creates a bucket named `minio-python-sdk-test-bucket`.
+- Uploads a file named `minio-python-sdk-test-file.bin` from `/tmp`.
+- Verifies the file was created using `mc ls`.
+
+```py
+# file_uploader.py MinIO Python SDK example
```

Comment:
Code feedback welcome. 

Attempted to be slightly more friendly for Windows users, but without fully getting into the cross-platform issues. Mainly by using `os.path.join()` instead of a string path, although a Windows user would still need to update `/tmp` to something appropriate.

I don't have a good way to actually test on Windows.

Reply:
As MS Windows and Python works with backslash (`/`) to refer file path on the system, I do not think it is necessary to cover MS Windows specifics here.

Reply:
After discussing with Docs team, decided to drop specific Windows mentions for now. Included a comment in the example about updating the path/filename, as a prompt for any reader who may need something different.